apps/web/src/app/api/activities/route.ts

```
@@ -0,0 +1,159 @@
+
+      case 'drive':
+        // All activity within a drive
+        if (!params.driveId) {
+          return NextResponse.json(
+            { error: 'driveId is required for drive context' },
+            { status: 400 }
+          );
+        }
+
+        // Verify user can view drive
+        const canViewDrive = await isUserDriveMember(userId, params.driveId);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)